astropy/cosmology/tests/test_core.py

```diff
@@ -306,7 +306,7 @@ def test_repr(self, cosmo_cls, cosmo):
 
         # parameters in string rep
         ps = {k: getattr(cosmo, k) for k in cosmo.__parameters__}
-        cps = {k: getattr(cosmo_cls, k) for k in cosmo.__parameters__}
```

Comment:
Apparently this line isn't used at all and should be removed!

Reply:
Didn't want to remove it in case creating it was part of the test. I will remove now

Reply:
Thanks!